src/tests/utils.rs

```diff
@@ -42,13 +42,14 @@ macro_rules! expect_event {
 }
 
 /// Generates random peer id.
-pub fn rand_peer_id() -> PeerId {
-    let (enc_pk, _enc_sk) = gen_encrypt_keypair();
+pub fn rand_peer_id() -> (PeerId, SecretEncryptKey) {
```

Comment:
`rand_peer_id` -> `rand_peer_id_and_enc_sk` as it was confusing to what sec-key was being returned in above (as you did not qualify the vars saying `peer_enc_sk` or `peer_sign_sk` but merely `peer_sk`. So if the function is appropriately named, it'll be less confusing.